ftp/src/main/scala/akka/stream/alpakka/ftp/impl/FtpIOGraphStage.scala

```diff
@@ -137,10 +138,18 @@ private[ftp] trait FtpIOSinkStage[FtpClient, S <: RemoteFileSettings]
       setHandler(
         in,
         new InHandler {
-          override def onPush(): Unit = {
-            write(grab(in))
-            pull(in)
-          }
+          override def onPush(): Unit =
+            try {
+              write(grab(in))
+              pull(in)
```

Comment:
Hmm should the `pull` be inside or outside the try/catch? we don't really expect that to throw, right?